code/controllers/subsystem/lobotomy_corp.dm

```diff
@@ -114,7 +114,7 @@ SUBSYSTEM_DEF(lobotomy_corp)
 	AdjustBoxes(amount)
 
 /datum/controller/subsystem/lobotomy_corp/proc/AdjustBoxes(amount)
-	current_box = clamp((current_box + amount), 0, box_goal)
+	current_box = current_box + amount
```

Comment:
Previous comment resolves this issue without allowing overflow. If you wish to allow overflow, I recommend reintroducing clamp in the form of "max".
```suggestion
	current_box = max(current_box + amount, 0)
```
So it can't somehow underflow.